resqpy/organize/organization_feature.py

```diff
@@ -1,15 +1,3 @@
-"""organize_old.py: RESQML Feature and Interpretation classes."""
```

Comment:
I strongly recommend preserving module docstrings - the "elevator pitch" of what the module does is useful for developers reading the source, and also key for the Sphinx documenation